packages/input-mask/README.md

```diff
@@ -91,6 +91,34 @@ return (
 
 In most cases you'll want to use `onInput` and `onPaste`.
 
+
+## Showing the pattern during input
+
+The placeholder attribute is nice, but it cannot show the pattern during input. In order to rectify this shortcoming, this package has a `createMaskPattern` export:
+
+```tsx
+import { createInputMask, createMaskPattern } from "@solid-primitives/input-mask";
+
+return <>
+  <style>{`
```

Comment:
I like including styles, but the online ones don’t have any syntax highlighting and look crammed together because of no formatting
Making a separate css code block could be clearer.

Reply:
Good point.